packages/ember-toucan-form/src/components/toucan-form.hbs

```diff
@@ -0,0 +1,20 @@
+<HeadlessForm
+  @data={{@data}}
+  @dataMode={{@dataMode}}
+  @validateOn={{@validateOn}}
+  @revalidateOn={{@revalidateOn}}
+  @validate={{@validate}}
+  @onSubmit={{@onSubmit}}
+  @onInvalid={{@onInvalid}}
```

Comment:
We can expose everything as we do here. But we _could_ also have opinions and not let users define everything on their own. Like for example we could define `@validateOn="focusout" and @revalidateOn="input"`, to make all forms behave consistently across our design system.

It's a design choice basically. Maybe worth discussing in one of the next jam sessions?

Reply:
For sure! I'll bring this up today!  Excellent call.

Reply:
Went ahead and added a different `validateOn` default, but we want to test this out in some real apps and get feedback before adjusting `revalidateOn`'s default